src/interpreter/PostInterpreter.ts

```diff
@@ -0,0 +1,70 @@
+import { CommonModel } from '../models';
```

Comment:
Hmm, do you think that makes sense? especially since it processes it as is using "simple" functions? 🤔

Reply:
Except that "interpretation" helper functions we have everything written in classes, so it's only a suggestion to have it also in class.

Reply:
but ok, we can stay with these functions.

Reply:
Fair point.

The reason I didn't wrap it in a class was that I didn't find the need. The Interpreter class was needed because it had to have access to a counter for anonym models as well as what schemas it already iterated. Passing these from function to function would have been annoying IMO.